sentry/src/test/java/io/sentry/util/InitUtilTest.kt

```diff
@@ -0,0 +1,99 @@
+package io.sentry.util
+
+import io.sentry.InitPriority
+import io.sentry.SentryOptions
+import kotlin.test.Test
+import kotlin.test.assertFalse
+import kotlin.test.assertTrue
+
+class InitUtilTest {
+
+    var previousOptions: SentryOptions? = null
```

Comment:
These could be private.

Also, should we re-init them for each test with `@Before`?